shell/platform/darwin/macos/framework/Headers/FlutterViewController.h

```diff
@@ -31,9 +31,19 @@ FLUTTER_DARWIN_EXPORT
 
 /**
  * The Flutter engine associated with this view controller.
+ *
+ * The engine is strongly referenced by the FlutterViewController, and weakly
+ * vice versa.
```

Comment:
The engine is strongly referenced by the FlutterViewController, and weakly referenced otherwise.